xtask/src/cli.rs

```diff
@@ -36,32 +36,48 @@ pub mod test {
         /// Skip building required binaries
         /// (sugondat-node, sugondat-shim, sov-demo-rollup and sov-cli)
         #[clap(default_value = "false")]
-        #[arg(long = "build-skip", value_name = "skip", id = "build.skip")]
+        #[arg(long = "skip-build", value_name = "skip", id = "build.skip")]
         pub skip: bool,
 
-        /// Don't print to stdout during the build process
-        #[arg(long = "build-quiet", value_name = "quiet", id = "build.quiet")]
-        pub quiet: bool,
+        /// Build process stdout and stderr are redirected into this file
+        #[arg(
+            long = "build-log-path",
+            value_name = "log-path",
+            id = "build.log-path"
+        )]
+        #[clap(default_value = "xtask/build.log")]
+        pub log_path: String,
```

Comment:
Given that these are all flattened into the same `Params`, how do you differentiate them?

Which one does `--log-path X` trigger?

Reply:
The trick with `value_name` and `id` makes the items look like `<prefix>-log-path`, where the prefix could be `zombienet`, `shim`, or `sovereign` to specify the path where you want to output the logs of the prefix process. This is a workaround, though, and the discussion on how the flags should look like can be found here: https://github.com/thrumdev/blobs/issues/224